src/Nelmio/Alice/Fixtures/Loader.php

```diff
@@ -256,8 +261,8 @@ protected function parseFile($filename)
     /**
      * builds a collection of fixtures
      *
-     * @param  array      $rawData
-     * @return array
+     * @param  array     $rawData
```

Comment:
I would put "see the example above to see the expected format of a raw data" and put an example under a `@example` tag. A good example would say far more than lengthy explanation